runtime/src/axum/mod.rs

```diff
@@ -101,6 +104,30 @@ impl Runtime for AxumWasm {
     ) -> Result<tonic::Response<Self::SubscribeLogsStream>, Status> {
         todo!()
     }
+
+    async fn stop(
+        &self,
+        request: tonic::Request<StopRequest>,
+    ) -> Result<tonic::Response<StopResponse>, Status> {
+        let request = request.into_inner();
+
+        let service_name = ServiceName::from_str(request.service_name.as_str())
+            .map_err(|err| Status::from_error(Box::new(err)))?;
+
+        let kill_tx = self.kill_tx.lock().unwrap().deref_mut().take();
+
+        if let Some(kill_tx) = kill_tx {
+            tokio::spawn(async move {
```

Comment:
Can we do this without spawning? Else this might need to be a `spawn_blocking` instead.

Reply:
I think we can do it without: https://docs.rs/tokio/latest/tokio/sync/oneshot/struct.Sender.html#method.send . I tested it now and it works. Thanks for the review!